prov/verbs/configure.m4

```diff
@@ -19,7 +19,28 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[],
 				[$verbs_PREFIX],
 				[$verbs_LIBDIR],
-				[verbs_ibverbs_happy=1],
+				[
+				AC_MSG_CHECKING(if libibverbs is linkable by libtool)
+				file=conftemp.$$.c
+				rm -f $file conftemp
+				cat > $file <<EOF
+/* Override any GCC internal prototype to avoid an error.
+   Use char because int might match the return type of a GCC
+   builtin and then its argument prototype would still apply.  */
```

Comment:
I'm probably being nit-picky, but this comment doesn't really apply here (I'm guessing you copied it from another autotools test program somewhere...?).  I.e., GCC will never have an internal prototype for ibv_open_device.